docs/source/executing-operations/subscription-callback-protocol.mdx

```diff
@@ -90,7 +92,7 @@ With initialization complete, the protocol commences its [main loop](#main-loop)
 
 The protocol's main loop remains active for the duration of the subscription. During the main loop, all of the following occur:
 
-- Every five seconds, **Emitter** must send a [`check` message](#check) to **Router** to confirm that **Router** is still listening.
+- Every `heartbeat_interval_ms` milliseconds (value coming from the initial payload sent in `extensions` from the **Router**), **Emitter** must send a [`check` message](#check) to **Router** to confirm that **Router** is still listening.
```

Comment:
maybe call out that we only do it IF heartbeat is enabled?

Reply:
Good call